python/cudf/cudf/pandas/scripts/run-pandas-tests.sh

```diff
@@ -138,7 +138,7 @@ and not test_eof_states"
 # TODO: Remove "not db" once a postgres & mysql container is set up on the CI
 PANDAS_CI="1" timeout 30m python -m pytest -p cudf.pandas \
     -v -m "not single_cpu and not db" \
-    -k "$TEST_THAT_NEED_MOTO_SERVER and $TEST_THAT_CRASH_PYTEST_WORKERS" \
+    -k "$TEST_THAT_NEED_MOTO_SERVER and $TEST_THAT_CRASH_PYTEST_WORKERS and not test_groupby_raises_category_on_category and not test_constructor_no_pandas_array and not test_is_monotonic_na and not test_index_contains and not test_index_contains and not test_frame_op_subclass_nonclass_constructor and not test_round_trip_current" \
```

Comment:
So these are still confirmed failing with now eagerly populating the class dict?

Reply:
yes, some of these segfaults and a few take quite a long time. The plan is to tackle each of them separately as follow-ups.